### PR TITLE
feat(api,ui): help text

### LINKED ISSUE #5178: feature request: add a "help" icon in UI
Hello,

Is it possible to add for example an icon " ? " in the navbar, with some useful information about contacts when we've an issue with CDS?

This can help people when CDS team contact informations have changed (like a Cisco room change), and when users have a red fish memory like me.

Thanks :)

### PR DESCRIPTION
**Pull request by yesnault**
close #5178

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: ui/src/app/views/navbar/navbar.html [hunk 1]
```diff
@@ -289,6 +289,19 @@
             </div>
         </ng-template>
 
+        <button *ngIf="help && help.content && help.content.length > 0"
```
Comment (sguiheux) [inline ui/src/app/views/navbar/navbar.html:292]: help?.content?.length > 0   (something like that)

File: ui/src/app/shared/toast/toast-http-error.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <div>
     <span *ngIf="toast.data.from">{{toast.data.from}}<br /></span>
-    <span *ngIf="toast.data.status === 500" style="font-size: 10px"><em>{{'toast_http_error_request_id' | translate}}:
-            {{toast.data.request_id}}</em></span>
+    <div *ngIf="toast.data.help && toast.data.help.length > 0">
```
Comment (sguiheux) [inline ui/src/app/shared/toast/toast-http-error.html:3]: toast?.data?.help?.length > 0 .   something like that

**Review by sguiheux — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14243.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

File: ui/src/assets/i18n/en.json [hunk 3]
```diff
@@ -653,7 +653,7 @@
   "timeline_filter_project": "Mute events on: ",
   "timeline_filter_updated": "Filter updated",
   "timeline_filter_project_select": "Select a project",
-  "toast_http_error_request_id": "Request ID",
+  "toast_http_error_request_id": "with this request ID:",
```
Comment (richardlt) [inline ui/src/assets/i18n/en.json:656]: I think we can let the old traduction for the case there is no message in the api config.

File: engine/api/api_routes.go [hunk 4]
```diff
@@ -148,6 +148,8 @@ func (api *API) InitRouter() {
 	r.Handle("/mon/errors/{uuid}", ScopeNone(), r.GET(api.getErrorHandler, NeedAdmin(true)))
 	r.Handle("/mon/panic/{uuid}", ScopeNone(), r.GET(api.getPanicDumpHandler, Auth(false)))
 
+	r.Handle("/help", ScopeNone(), r.GET(api.getHelpHandler, Auth(false)))
```
Comment (fsamin) [inline engine/api/api_routes.go:151]:
```suggestion
	r.Handle("/help", ScopeNone(), r.GET(api.getHelpHandler))
```

Comment (fsamin) [reply engine/api/api_routes.go:151]: i would keep authentication, if the help message contains contact information, we may only want to give them to registered users

File: ui/src/assets/i18n/fr.json [hunk 5]
```diff
@@ -651,7 +651,7 @@
   "timeline_loading": "Chargement du fil d'actualité...",
   "timeline_no": "Le fil d'actualité est vide",
   "timeline_title": "Fil d'actualité",
-  "toast_http_error_request_id": "Identifiant de la requête",
+  "toast_http_error_request_id": "avec cet identifiant de requête:",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json:654]: i prefer the old message

**Review by fsamin — CHANGES_REQUESTED** [review]

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14251.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package and Analyze
  * Docker ✔ 
  * Sonar ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘  
  * Standalone tests 1 -   
  * Standalone tests 2 -   
  * Standalone tests 3 -

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14254.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package and Analyze
  * Docker ✔ 
  * Sonar ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘  
  * Standalone tests 1 -   
  * Standalone tests 2 -   
  * Standalone tests 3 -

**Review by richardlt — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**